app/collins/util/parsers/LshwParser.scala

```diff
@@ -79,6 +82,13 @@ class LshwParser(txt: String) extends CommonParser[LshwRepresentation](txt) {
       Cpu(cores, threads, speed, asset.description, asset.product, asset.vendor)
   }
 
+  val gpuMatcher: PartialFunction[NodeSeq, Gpu] = {
+    case n if ((n \ "@class" text) == "display" && (n \\ "vendor" text).contains("NVIDIA Corporation")) => {
```

Comment:
yikes, we cant have this. We should check a config parameter that has a list of valid GPU vendors instead